src/config/ConfigService.tsx

```diff
@@ -4,7 +4,7 @@ import { createGenericContext } from '../services/GenericContext';
 import { UppercaseObjectKeys, ValidateKeysTuple } from '../types/TypeTransformUtil';
 import { snakeToCamelCase } from '../util/Notation/Notation';
 
-const BACKEND_URL = process.env.REACT_APP_BACKEND_URL ?? 'http://localhost:5000';
+const BACKEND_URL = process.env.REACT_APP_BACKEND_URL ?? 'https://localhost:8443';
```

Comment:
Why this has been changed here ? According to the docs the UI should talk to http backend, see https://yaptide.github.io/deploy/

Reply:
```suggestion
const BACKEND_URL = process.env.REACT_APP_BACKEND_URL ?? 'http://localhost:5000';
```